src/printer.js

```diff
@@ -35,6 +35,8 @@ const {
   getText,
 } = require('./utils');
 
+const supportedStyleLangValues = ['css', 'sass'];
```

Comment:
Shouldn't it be "scss" not "sass"? since scss = sass w/ css syntax vs scss = sass w/ indented syntax

Reply:
Oh, sorry! Sometimes mistakes happen.